.github/workflows/scripts/ti_build/compiler.py

```diff
@@ -33,7 +33,7 @@ def setup_clang(as_compiler=True) -> None:
     """
     u = platform.uname()
     if u.system == "Linux":
-        for v in ("-15", "-14", "-13", "-12", "-11", "-10", ""):
+        for v in ("-21", "-20", "-19", "-18", "-17", "-16", "-15", "-14", "-13", "-12", "-11", "-10", ""):
```

Comment:
This looks scary. What are your thought on how we can test this?

Reply:
@duburcqa Thoughts on this? I'm tempted to accept the change, but request that -14, -13, -12, -11, -10 are removed. Thoughts?